packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BubbleGraph/General/MultipleDatasets.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
 import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
-import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
 import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
 import getSimpleAxisData from '@cerner/terra-graphs-docs/src/example-datasets/graphConfigObjects/Bubble/simpleAxisData';
```

Comment:
Looks like you forgot make changes to import here to import getSimpleAxisData from '@cerner/terra-graphs-docs/**lib**/example-datasets/graphConfigObjects/Bubble/simpleAxisData'

Also, there are other places which need this change.

Reply:
fixed here: 13c8a20433e8369c6ac78082cbd46db3fe92a51c